test/library/draft/arrow/ArrowParquet.chpl

```diff
@@ -0,0 +1,60 @@
+module ArrowParquet {
+  require "ArrayDecl.chpl", "ParquetDecl.chpl", "ArrowRecordDecl.chpl";
+  use ArrayDecl, ParquetDecl, ArrowRecordDecl;
+  use SysCTypes, CPtr;
+
+  proc writeTableToParquetFile(table: GArrowTable, path: string) {
+    var error: GErrorPtr;
+    var writer_properties: GParquetWriterProperties = gparquet_writer_properties_new();
+    var writer: GParquetArrowFileWriter = gparquet_arrow_file_writer_new_path(
+                                                  garrow_table_get_schema(table),
+                                                  path.c_str(), 
+                                                  writer_properties,
+                                                  c_ptrTo(error));
+  if(isNull(writer)){
+      g_print("failed to initialize writer: %s\n", error.deref().message);
+      g_error_free(error);
+      exit(EXIT_FAILURE);
+    }
+    var success: gboolean = gparquet_arrow_file_writer_write_table(writer,
+                                                      table ,
+                                                      10 : guint64,
```

Comment:
Do we want this to be hard coded 10? I see that it is chunk size in the documentation (https://arrow.apache.org/docs/c_glib/parquet-glib/GParquetArrowFileWriter.html#gparquet-arrow-file-writer-write-table), but I am not exactly sure what that means without looking into it further.

Reply:
We definitely don't want this to be hardcoded. We want some intelligent way to figure this and also give the user an option to pass it maybe.